stdlib/os/__init__.pyi

```diff
@@ -365,7 +365,9 @@ class stat_result(structseq[float], tuple[int, int, int, int, int, int, int, flo
     if sys.version_info >= (3, 12) and sys.platform == "win32":
         @property
         @deprecated(
-            "Use st_birthtime instead to retrieve the file creation time. In the future, this property will contain the last metadata change time."
+            """\
+Use st_birthtime instead to retrieve the file creation time. \
+In the future, this property will contain the last metadata change time."""
```

Comment:
I'm not a big fan of this:
- For someone not deeply familiar with details of Python's syntax, this is much more magical than the previous long one-liner.
- This is fragile and easy to mess up. Move the backslash or add indentation, basically touch it in any way, and it's not the same string anymore. Some changes affect `inspect.cleandoc()` while others do not.

If we're extremely concerned about long strings for some reason, can't we just run black with `--experimental-string-processing` (or a non-deprecated alternative if it exists and works)?

Reply:
We use multiline strings and multiline implicit concatenations for a handful lot of `@deprecated`
![image](https://github.com/user-attachments/assets/2e7e3f4a-af7a-4fae-8693-7f12548df8d4)

If you have concerns about using multiline strings here, maybe we could review our usage and change the ones in pywin32 to multiline ISC.

I'm really not a fan of multiline ISC, as they look like multiple arguments, rather than the same strings. But with the tooling we have in place, ISC errors can be caught, whilst, to your point, accidental whitespace changes in multiline strings can't.

Reply:
I'm no longer convinced that multiline strings are the worst solution. Each alternative has its own problems:
- multiline strings: need to look at indentations carefully (indentation is included in the string)
- ISC: need to pay attention to commas and middle spaces (if you want `"foo bar"` then these are all wrong: `"foo" "bar"`, `"foo " " bar"`, `"foo ", "bar"`)
- long lines: not nice in editors unless you enable word wrap

Reply:
Just as a note:
`"foo " " bar"` can be caught by https://docs.astral.sh/ruff/rules/single-line-implicit-string-concatenation/ (multiline ISC is one thing, but there's really no reason for single line ones). Unless you meant multiline ISC, but were just being concise in your example

`"foo ", "bar"` should be caught by mypy and pyright

Reply:
@Akuli Are you fine with the correct state of this PR or would you like to discuss this further ?